### DISCUSSION TITLE
Hide pre-release release

### DISCUSSION THREAD

**Discussion by Dongdarealest** [community/community#18659] [Other Feature Feedback, Questions, & Ideas] [answered]
I have some projects starred (Ionic Framework, Next.js, Remix) that release a prolific number of pre-release that clog up my feed. I'd love to have an option to hide pre-release releases and just show major.

**Comment by drewlustro** [comment]
Yes please. It would be great to also filter pre-releases on the Releases tab.

**Comment by jonaseberle** [comment]
I think this is still a usability issue with releases.

In projects that actively use pre-releases (which is great IMHO) it is hard for a "normal" user to find the stable releases on /releases.

**Comment by mytlogos** [comment]
Duplicate of https://github.com/orgs/community/discussions/18499 ?

**Reply by mytlogos** [reply, marked as answer]
For just viewing the releases (not feed): https://github.com/orgs/community/discussions/6108 
Answer from this was using the filter `prerelease:false` in the search bar.

**Reply by Netail** [reply]
Why is this marked as the answer? The original question was about the feed and this answer is about the release page. There's still no solution to filter pre-releases out from the home feed

**Comment by yeelam-gordon** [comment]
Any follow up on this after like 4 years?
